source/css/style.css

```diff
@@ -243,6 +243,11 @@ h1, h2, h3, h4, h5, h6 {
 .paddings{
 	padding: 70px 0;
 }
+@media (max-width: 991px) {

```

Comment:
The changes were made in whichever existing file the class was in, with new additions being opted for the scss file. The .paddings class already existed in the .css file, so that was where I left the style. I'm unsure of why this was originally setup with css and scss files